charts/spiffe-helper-csi-driver/README.md

```diff
@@ -0,0 +1,48 @@
+# spire-helper-csi-driver
+
+![Version: 0.1.0](https://img.shields.io/badge/Version-0.1.0-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square) ![AppVersion: 1.7.2](https://img.shields.io/badge/AppVersion-1.7.2-informational?style=flat-square)
+
+A Helm chart to install the SPIFFE HELPER CSI Driver.
+
+**Homepage:** <https://github.com/spiffe/helm-charts/tree/main/charts/spire>
+
+## Maintainers
+
+| Name | Email | Url |
+| ---- | ------ | --- |
+| marcofranssen | <marco.franssen@gmail.com> | <https://marcofranssen.nl> |
+| kfox1111 | <Kevin.Fox@pnnl.gov> |  |
+| faisal-memon | <fymemon@yahoo.com> |  |
+| edwbuck | <edwbuck@gmail.com> |  |
+
+## Source Code
+
+* <https://github.com/spiffe/helm-charts/tree/main/charts/spiffe-helpe-csi-driver>
+
+## Prereqs:
+
+Your cluster needs to have Kyverno installed. You can do that by running something like the following:
+
+```
+helm upgrade --install --create-namespace kyverno kyverno -n kyverno --repo https://kyverno.github.io/kyverno/ --version 3.1.1
```

Comment:
Let's not slip in a policy engine as an out-of-scope "extra" on a commit intended to add a CSI driver.